src/components/ModalBilling/ModalBilling.tsx

```diff
@@ -177,6 +183,14 @@ class ModalBilling extends React.Component<Props, State> {
                 type="button"
                 className={'modalButton--filled'}
                 onClick={this.showPaymentModal}
+                disabled={
+                  !this.state.name ||
```

Comment:
@stanleyzhengnyc brings up a good point in that we can rip this out. Could you work with @kevinjtsai on moving the email field to the CC part of the flow?

Reply:
@jtmckibb removed modal billing & added email and name fields to payment modal

Reply:
Looks good!